.github/workflows/plugin_install.yml

```diff
@@ -3,7 +3,7 @@ name: Plugin Install
 on: [push, pull_request, workflow_dispatch]
 
 env:
-  OPENSEARCH_VERSION: 2.4.1
```

Comment:
Oops. Sorry for the spam as well--this formatting was going to prevent other repos from being able to use the action since they cannot assemble in the called repo's context. :/